src/NuGet.Core/NuGet.Test.Utility/TestPackages.cs

```diff
@@ -539,70 +543,70 @@ public static Stream GetTestPackageWithContentXmlFile()
             return stream;
         }
 
-        public static FileInfo GetPackageWithSHA512AtRoot(string path, string packageId, string packageVersion)
+        public static async Task<FileInfo> GetPackageWithSHA512AtRoot(string path, string packageId, string packageVersion)
```

Comment:
Spotted the only consumer in a test that was async before the change. Besides GeneratePackageAsync method hides async stream writing so that entire call chain has to be async.